cmake/package_search/Finddoca.cmake

```diff
@@ -30,112 +27,119 @@ endif()
 
 # Now find DPDK
 find_package(libdpdk QUIET)
```

Comment:
Remark: This should probably not be quiet, since we'll fail (?) later if we don't find it.

Reply:
Fixed